include/mender-utils.h

```diff
@@ -43,7 +43,9 @@ extern "C" {
 /**
  * @brief A utility macro to make marking unused arguments less noisy/disruptive
  */
+#ifndef ARG_UNUSED
 #define ARG_UNUSED __attribute__((unused))
+#endif
```

Comment:
Hmm, does it work? The definition there (below) doesn't look like something that would work the same way. At least not in an argument/parameter list.
```
#define ARG_UNUSED(x) (void)(x)```

Reply:
It works and it is broadly used in the kernel code.